src/ui/public/styles/base.less

```diff
@@ -656,4 +660,14 @@ fieldset {
   }
 }
 
+.page-row {
```

Comment:
I wonder if this shouldn't be in Kibana itself? I suspect Monitoring's cluster overview would benefit from this as well.